course_catalog/etl/video.py

```diff
@@ -0,0 +1,62 @@
+"""video catalog ETL"""
+
+from django.conf import settings
+from googleapiclient.discovery import build
+import yaml
+import requests
+
+YOUTUBE_API_SERVICE_NAME = "youtube"
+YOUTUBE_API_VERSION = "v3"
+
+
+def get_youtube_client():
+    """Client for google api"""
```

Comment:
Could you expand this docstring a bit? I don't think it's clearly outlined in any of our onboarding docs, but we generally follow the google style guide to define arguments and return values: https://google.github.io/styleguide/pyguide.html?showone=Comments#38-comments-and-docstrings